Lib/tkinter/__init__.py

```diff
@@ -1827,18 +1827,24 @@ def _configure(self, cmd, cnf, kw):
     # These used to be defined in Widget:
 
     def configure(self, cnf=None, **kw):
-        """Configure resources of a widget.
+        """Query or modify the configuration options of the widget.
```

Comment:
```suggestion
        """Query or modify the configuration options of widget *self*.
```
The expand text below is very nice.

Reply:
`*` and other markup is not used in docstrings. At least not in this file.